spec/features/admin/sign_in_as_admin_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'features/support/sign_up_helpers'
+require 'features/support/not_signed_in'
+
+describe 'signing as an admin user' do
+  let!(:user) { create(:user, :confirmed, admin: true) }
+
+  context 'when visiting the home page' do
+    before do
+      5.times { create(:organisation) }
+      sign_in_user user
+      visit root_path
+    end
+
+    it 'redirects to the organisations page' do
+      expect(page.current_path).to eq(organisations_path)
```

Comment:
I wouldn't test this, I'd be happy relying on the test below (checking the content of the page) as sufficient for proving this feature is working.

This test feels brittle to me, as it's tied to an implementation detail (the content we're looking for being at a particular URL).